### PR TITLE
Mergable version of oc-161 branch

### PR DESCRIPTION
**Pull request by pczeeuw**
Added new Technical Chapter oc-161, changed severals pages as described in ox-162, restructure domain oc-163, added feedback from Sander J

### CODE REVIEW

File: Userguide/CreateWebApp.md [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
-- The request should have a header where an ApplicationName, UserName and OrganisationId are defined. At the moment User and App name are not required for the Platform (Besides Audability). The OrganisationId, however, must match a known organisation within the Platform. This organisation must have the right authortities to make a certain request.
-- Furthermore, the Request must be signed with a certificate from the Organisation with the OrganisationId in the header.
+- In the WSDL and xsd files of the Platform the Requests and it's objects are defined.
+- Fill in the parameters once you have an empty soap request for a certain function. The restrictions/requirements are defined in the WSDL files.
+- The request should have a header where an ApplicationName, UserName and OrganisationId are defined. At the moment User and App name are not used in the Platform (Except fir audability and logging). The OrganisationId, however, must match a known organisation within the Platform. This organisation must have the right authortities to make a certain request.
```
Comment (Sander3003) [inline Userguide/CreateWebApp.md]: 'Except fir' is like likely 'Except for'

**Comment by Sander3003** [comment]
Looks good to me!

File: Architecture/Technical-overview/ProtocolLayer.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+## Protocol Adapters
+
+The Protocol Adapters are responsible for the actual communication to and from a device. They usually operate in a certain domain, and might use common/generic functiuons from the platform.
```
Comment (smvdheijden) [inline Architecture/Technical-overview/ProtocolLayer.md]: functiuons = functions

File: Architecture/Technical-overview/ProtocolLayer.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+The Open Smart Grid Platform currently has the following protcol adapters:
+- Protocol-Adapter-DLMS: Smart Metering 
+- Protocol-Adapter-IEC61850: Public Lighting
```
Comment (smvdheijden) [inline Architecture/Technical-overview/ProtocolLayer.md]: IEC61850 protocol is used for Public Lighting and Microgrids

File: Architecture/Technical-overview/Technologystack.md [hunk 4]
```diff
@@ -1,60 +1,60 @@
...
+- [SoapUI](https://www.soapui.org/): Functional testing tool for API testing
+- [Mockito](http://mockito.org/): Unit testing
+- [JUnit](http://junit.org/): Unit testing
+
+This Table presents an overview of the components and the most important technological choices per component.
+
+| **Component** | **Technology** |
+| --- | --- |
+| OSGP Platform | Java, Spring Framework, Hibernate, Netty |
+| Demo applicatie | Java, Spring Framework, Spring MVC |
```
Comment (smvdheijden) [inline Architecture/Technical-overview/Technologystack.md]: applicatie = application
Is this application already available in the open source repositories?

File: Architecture/Technical-overview/Technologystack.md [hunk 5]
```diff
@@ -1,60 +1,60 @@
...
+| Web services | SOAP, WSDL |
+| OSLP Protocol | Google Protocol Buffers |
+
+
+| **Component (not open source)** | **Technology** |
+| --- | --- |
+| OSGP Management application | Java, Spring Framework, Spring MVC |
+| Net-Management application | Java, Spring Framework, JAX-RS, AngularJS |
 | Liander Installatie application | Java, Spring Framework, JAX-RS, AngularJS |
```
Comment (smvdheijden) [inline Architecture/Technical-overview/Technologystack.md]: Should these applications be mentioned here in the open source documentation?

File: Architecture/Technical-overview/WebServicesLayer.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+- Core - osgp-adapter-ws-core: Contains the Core(common) web services.
+- Admin - osgp-adapter-ws-admin: Contains the Admin web services.
+- Shared - osgp-adapter-ws-shared: Contains shared endpoints (such as header authorization)
+- Database - osgp-adapter-ws-db: Contains repositories for persistence.
+
+#### Domain
+- Public Lighting - osgp-adapter-ws-publiclighting: Contains the Public Lighting web services.
+- Smart Metering - osgp-adapter-ws-smartmetering: Contains the Smart Metering web services.
+- Tariff Swithcing - osgp-adapter-ws-tariffswitching: Contains the Tariff Switching web services.
+
```
Comment (smvdheijden) [inline Architecture/Technical-overview/WebServicesLayer.md]: +Microgrids

File: Architecture/Technical-overview/Technologystack.md [hunk 7]
```diff
@@ -1,60 +1,60 @@
...
+- [Puppet](https://puppet.com/): Application for Automatically delivering, operating and securing your infrastructure 
+
+### Development
+
+- [Bower](http://bower.io/): Package manager for Javascript packages. Web sites are made of lots of things - frameworks, libraries, assets, utilities, and rainbows. Bower manages all these things for you.
+- [Eclipse](https://eclipse.org/): IDE for developing software.
+- [FileZilla](https://filezilla-project.org/): FTP application.
+- [Git](https://git-scm.com/): Version control system.
+- [NodeJS](https://nodejs.org/): Tooling suite with various Javascript tools.
+- [Notepad++](https://notepad-plus-plus.org/): A free source code editor.
```
Comment (smvdheijden) [inline Architecture/Technical-overview/Technologystack.md]: I wouldn't mention Notepad++, on different platforms other text editors might be used, like for example vim, gedit, kate...

File: Architecture/Technical-overview/Technologystack.md [hunk 8]
```diff
@@ -1,60 +1,60 @@
...
+- [NPM](https://www.npmjs.com/): Package manager for the NodeJS Javascript applications.
+- [Putty](http://www.chiark.greenend.org.uk/~sgtatham/putty/): A free and open-source terminal emulator, serial console and network file transfer application.
+- [SourceTree](https://www.sourcetreeapp.com/): SourceTree is a powerful Git and Mercurial desktop client for developers on Mac or Windows.
```
Comment (smvdheijden) [inline Architecture/Technical-overview/Technologystack.md]: On linux platforms other tools might be used, for example just git from the command line, or clients like git cola.

File: Userguide/Deployment.md [hunk 9]
```diff
@@ -3,3 +3,7 @@
 Hosting the open smart grid platform in the cloud is possible, as well as on premises.
 
  ![How open smart grid platform components relate to each other](./Typical_Opensmartgridplatform_setup.png "Typical platform setup including webapplications and devices")
+
+### Maintenance
+
+There's not much maintenance that needs to be performed. Archiving some old log files, checking up on available disk space and creating a backup of the databases. Looking into the queues to see if there are not handled messages in dead letter queue.
```
Comment (smvdheijden) [inline Userguide/Deployment.md]: if there are no messages in the dead letter queue.

**Pull request merged**